### DISCUSSION TITLE
Problems with installing OpenGL for Davinci

### DISCUSSION THREAD

**Discussion by meaganmargaret** [ROCm/ROCm#3338] [Q&A] [answered]
So, I have spent quite a few hours trying to get Davinci Resolve to run and I've had no luck.

From inxi -Gxxxz

Graphics:  Device-1: Advanced Micro Devices [AMD/ATI] vendor: ASRock driver: amdgpu v: kernel bus ID: 03:00.0 
           chip ID: 1002:73bf 
           Display: x11 server: X.Org 1.20.13 driver: amdgpu,ati unloaded: fbdev,modesetting,radeon,vesa 
           resolution: 3840x2160~60Hz 
           OpenGL: renderer: AMD Radeon RX 6900 XT (navi21 LLVM 14.0.5 DRM 3.42 5.15.0-107-generic) 
           v: 4.6 Mesa 22.2.0-devel (git-e8fc5cc 2022-06-22 focal-oibaf-ppa) direct render: Yes 

When I do a amdgpu install, and specify the --opengl -rocr parameter, it always says it completes, but stops and complains about:
E: Unable to locate package clinfo-amdgpu-pro
E: Unable to locate package opencl-legacy-amdgpu-pro-icd

And Davinci won't load because, I think, OpenGL (rocr) just doesn't get installed.

Any help out there?

**Comment by taylding-amd** [comment, marked as answer]
Hi @meaganmargaret, You can find detailed documentation on [how to install AMDGPU](https://amdgpu-install.readthedocs.io/en/20.40/install-script.html). Please note that OpenGL is included as a default component during the installation.

For any additional questions related to drivers, you may find better assistance in the [AMD Community](https://community.amd.com/), where experts will be happy to assist you.
